# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332345611 18000
#      Wed Mar 21 11:00:11 2012 -0500
# Node ID 8220d6713ea21d8a119911ceed844e89f9fbcca4
# Parent  9328e5d72eb288f306a95050929426ad9a6724d2
gklib: update header for __thread fix

diff -r 9328e5d72eb2 -r 8220d6713ea2 headers/gk_externs.h
--- a/headers/gk_externs.h	Tue Mar 20 11:59:44 2012 -0500
+++ b/headers/gk_externs.h	Wed Mar 21 11:00:11 2012 -0500
@@ -12,14 +12,19 @@
 
 
 /*************************************************************************
 * Extern variable definition. Hopefully, the __thread makes them thread-safe.
 **************************************************************************/
+
+/* include the definition of metis__thread; it is assumed that
+   gklib_tls.h will be installed 'globally' i.e. into $PREFIX */
+#include <gklib_tls.h>
+
 #ifndef _GK_ERROR_C_
 /* declared in error.c */
-extern __thread int gk_cur_jbufs;
-extern __thread jmp_buf gk_jbufs[];
-extern __thread jmp_buf gk_jbuf;
+extern metis__thread int gk_cur_jbufs;
+extern metis__thread jmp_buf gk_jbufs[];
+extern metis__thread jmp_buf gk_jbuf;
 
 #endif
 
 #endif
